Vagrantfile

```diff
@@ -13,9 +13,9 @@ SRCPATH="/opt/gopath"
 ARCH=`uname -m | sed 's|i686|386|' | sed 's|x86_64|amd64|'`
 
 # Install Prereq Packages
-sudo apt-get update
-sudo apt-get upgrade -y
-sudo apt-get install -y build-essential curl git-core libpcre3-dev mercurial pkg-config zip
+sudo DEBIAN_FRONTEND=noninteractive apt-get update
```

Comment:
You could export `DEBIAN_FRONTEND=noninteractive` this once, I believe.

In fact, for ultimate unattended experience, you might want this:

```
export DEBIAN_PRIORITY=critical
export DEBIAN_FRONTEND=noninteractive
export DEBCONF_NONINTERACTIVE_SEEN=true
```